src/main/java/mate/hw13/sql/CrudOperations.java

```diff
@@ -0,0 +1,49 @@
+package mate.hw13.sql;
+
+import util.JdbcConnectionUtil;
+
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class CrudOperations {
+
+    public static void createOperation() {
+        Statement statement = JdbcConnectionUtil.getConnection();
+        String query = "insert into developers(name, age, sex, salary) values ('anja', 21, 'female', 20)";
+        try {
+            statement.execute(query);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void readOperation(){
```

Comment:
why read method returns nothing
and dont forget about java style
for method names better use verb or verb phrases